src/test/shared/extensionUtilities.test.ts

```diff
@@ -23,4 +23,25 @@ describe('extensionUtilities', () => {
             assert.strictEqual(safeGet(undefined as Blah | undefined, x => x.someProp), undefined)
         })
     })
+
+    describe('convertPathTokensToPath', () => {
```

Comment:
Please add a test where the input text does not contain the token of interest

Reply:
✔️ Added to the `createWelcomeWebview` tests